koku/api/urls.py

```diff
@@ -335,6 +337,8 @@
         ),
         name="reports-openshift-azure-instance-type",
     ),
+    path("reports/minimal-reports/", MinimalReportView.as_view(), name="minimal-reports"),
```

Comment:
@adberglund thoughts on the API here?
Naming is hard I know. Do we feel that `reports/minimal-reports/` is redundant?
Do we want this to live under `reports`?
Could do something like `ingress/minimal/`. If we are happy with the above ok. I just want to make sure we've thought it through.

Reply:
I also hate reports reports, I went ahead and made it ingress/minimal-reports/ for now.

Reply:
I know right now we're tackling the idea of a filtered subset of data, but we could keep it broader and remove `minimal` from external reference. Something like `ingress/reports/` ?